### PR TITLE
YottaDB support; autoconfig support; absorb OSEHRA changes; remove box-vol change

### PR DESCRIPTION
**Pull request by shabiel**

### CODE REVIEW

File: GTM/gtminstall_SHA1 [hunk 1]
```diff
@@ -1 +1 @@
-229b360372fb275611980b6e61f1b25347cbaa3d  gtminstall
+630cf46d5d4d0dc55230837ee327d227269a70fc gtminstall
```
Comment (ChristopherEdwards) [inline GTM/gtminstall_SHA1:1]: should this be renamed ydbinstall? instead of forcing everything to gtminstall?

File: GTM/install.sh [hunk 2]
```diff
@@ -108,7 +127,11 @@ fi
 # Accept most defaults for gtminstall
 # --ucaseonly-utils - override default to install only uppercase utilities
 #                     this follows VistA convention of uppercase only routines
-./gtminstall --ucaseonly-utils --installdir /opt/lsb-gtm/"$gtm_ver"_"$gtm_arch" $gtm_ver
+if [ "$installYottaDB" = "true" ] ; then
+    ./gtminstall --ucaseonly-utils --installdir /opt/yottadb/"$gtm_ver"_"$gtm_arch" $gtm_ver
```
Comment (ChristopherEdwards) [inline GTM/install.sh:131]: I'd prefer using ydbinstall instead of gtminstall.

File: README.md [hunk 3]
```diff
@@ -10,9 +10,6 @@ This has only been tested using:
 ## Pre-requisites
 A working [Docker](https://www.docker.com/community-edition#/download) installation on the platform of choice.
 
-## Note
-You cannot use docker exec -it osehravista bash to get access to the mumps prompt. This is likely due to how docker permission schemes for shared memory access works. Instead always gain access via the ssh commands below
```
Comment (ChristopherEdwards) [inline README.md]: Why is this deleted? I've verified that the issue still exists, though I have more information: This only occurs if someone else is accessing the instance at the same time (CPRS, etc.) as you do a docker exec -it osehravista bash. The workaround is to ssh into the instance after running the docker exec -it osehravista command.

**Comment by shabiel** [comment]
Okay. I couldn't replicate it; but as you verified that it still exists,
then I can undo that change.

As to ydbinstall--A good idea, but doing that means that I have to re-test
with two scenarios--which takes a while. What do you think? I am inclined
to postpone to a future commit.

--Sam

**Comment by ChristopherEdwards** [comment]
@shabiel I've created a new issue to deal with the rename. I'll wait for the commit to come in to re-add the removed comment in the readme.

**Comment by shabiel** [comment]
​The comment has been re-added and pushed.
